doc/events/alarm.md

```diff
@@ -0,0 +1,17 @@
+---
+module: [kind=event] alarm
+see: os.setAlarm To start an alarm.
+---
+
+The @{timer} event is fired when an alarm started with @{os.setAlarm} completes.
+
+## Return Values
+1. @{number}: The ID of the alarm that finished.
+
+## Example
+Starts a timer and then prints its ID:
+```lua
+os.setAlarm(os.time() + 0.05)
+local event, id = os.pullEvent("alarm")
+print("Alarm with ID " .. id .. " was fired")
```

Comment:
Is it worth doing this in a `while` loop and checking the IDs match? 

Obviously no other `alarm` events are going to be queued when running this, but just to demonstrate "good practice" or something.